documentation/Developer-Documentation/Contributing-Guidelines.md

```diff
@@ -0,0 +1,61 @@
+# Contributing Guidelines
+
+The BetonQuest organisation recommends IntelliJ as the preferred IDE.
```

Comment:
I think https://www.jetbrains.com/idea/ is suited better as it shows why the tool is great instead of just giving the user the download page that is filled with the monetary aspect of Community vs. Ultimate.